src/Console/ConfigurationResolver.php

```
@@ -588,6 +588,12 @@ private function parseRules()
             return $this->getConfig()->getRules();
         }
 
+        $jsonRules = json_decode($this->options['rules'], true);
+
+        if (is_array($jsonRules)) {
```

exit on invalid user input instead of silently going forward